playground/infrastructure/helper.py

```diff
@@ -44,29 +54,221 @@ def find_examples(work_dir: str) -> List[Example]:
         categories:
             - category-1
             - category-2
+    If some example contain beam tag with incorrect format raise an error.
 
     Args:
         work_dir: directory where to search examples.
+        supported_categories: list of supported categories.
 
     Returns:
         List of Examples.
     """
-    # TODO [BEAM-13135] Implement
-    pass
+    has_error = False
+    examples = []
+    for root, _, files in os.walk(work_dir):
```

Comment:
have you ran this? do you know how long it takes to run? just curious

Reply:
Yes. I have run a little test to check how long this method works. Now it takes about 16 seconds.